LeetCode/array.md

```diff
@@ -0,0 +1,133 @@
+#Array
```

Comment:
Your Pull Request title: Array

Check result:

* Pull Request title must start with prefix in ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER', 'REQUESTS', 'ITERATOR']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title action must start with ['Added', 'Deleted', 'Refactored', 'Moved', 'Fixed']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']
* Pull Request title must contain group number in ['1021', '1022']